packages/app/src/components/Visualization/StartScreen.js

```diff
@@ -1,17 +1,18 @@
 import React, { useEffect, useState } from 'react'
 import { connect } from 'react-redux'
 import i18n from '@dhis2/d2-i18n'
+import PropTypes from 'prop-types'
+import { withStyles } from '@material-ui/core/styles'
```

Comment:
Can this be also converted or is it out of scope for this PR?

Reply:
Haha whoops, just my OCD that forced me to reorder the imports 😄 but there's a separate PR for this coming